site/templates/collection/carousel/item/buttons.jet

```diff
@@ -0,0 +1,26 @@
+{{import "./extras.jet"}}
+
+{{block carouselItemButtons()}}
+  {{slug := .Slug}}
+  {{isFilm := .ItemType == "film"}}
+  {{hasTrailer := (isset(.InnerItem["Trailers"]) && len(.InnerItem.Trailers) > 0) || (.InnerItem["PromoURL"] && len(.InnerItem.PromoURL) > 0)}}
```

Comment:
What is `PromoUrl`?

Reply:
From [here](https://github.com/shift72/core-template/blob/e7ec4c78292b8917836e2f140927d1ee5f74c8fa/site/templates/collection/carousel_item.jet#L9).  I don't see it on the kibble models though... maybe it's a legacy thing?